packages/ui-helpers/src/createTestId.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-console */
+const DOUBLE_INDEX_CHANGE = 2;
+
+type CreateTestIdType = (args: (number | string)[]) => string;
+
+const createTestId = (blockName: string): CreateTestIdType => {
+    const stringPatter = /^[a-zA-Z]+$/;
```

Comment:
```suggestion
    const stringPattern = /^[a-zA-Z]+$/;
```
И лучше вынести наверх.

Reply:
может быть такая ситуация. Вложенность компонентов
`Wrapper -> Carousel -> CarouselItem -> CarouselItemButton`
В CarouselItem передается пропс testId="wrapper-carousel-carouselItem[1]" и внутри он берет этот пропс прогоняет повторно через эту функцию и добавляет button. Так что такой паттерн не подходит.